### PR TITLE
LL

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

File: test/findLoop.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+n['next']=z;
+const test=(obj1,obj2,TF)=>{
+  if (solution(obj1,obj2)==TF){
+    console.log("expected output");
+  }
+  else{
+    console.log("Unexpected output");
+  }
+}
+test(r,r,false);
```
Comment (evo777) [inline test/findLoop.js:20]: Shouldn't this be `true`?

Comment (saumya665) [reply test/findLoop.js:20]: r is not a circular linked list, so output is false.

File: solutions/findLoop.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const LL=(obj,refObj)=>{
+  if(obj.next===refObj){
+    return true;
+  }
+  if(!obj.next){ return false}
+  return LL(obj.next,refObj);
+  }
+module.exports = LL;
+
+//console.log(LL(a,a));
```
Comment (seemcat) [inline solutions/findLoop.js]: Remove commented code.

File: solutions/findLoop.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const b = {val:2};
```
Comment (seemcat) [inline solutions/findLoop.js]: remove since you already have a test file

File: test/findLoop.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+const a ={val:2};
+a['next']=b;
+b['next']=a;
+
+const r = {val:5};
+
+const z={val:2};
+const n = {val:1};
+n['next']=z;
+const test=(obj1,obj2,TF)=>{
```
Comment (seemcat) [inline test/findLoop.js]: what is TF? How about staying consistent and just naming it result

**Review by seemcat — CHANGES_REQUESTED** [review]

**Review by seemcat — APPROVED** [review]

**Pull request merged**